module/Motion/ToCross.cpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file ToCross.cpp
+ * @brief 交点サークル間移動のクラス

```

Comment:
```suggestion
 * @file CrossToCross.cpp
 * @brief 交点から交点までライントレースするクラス

```
交点サークル間って言葉ちょっとわかりにくいかな